mautrix_signal/commands/util.py

```diff
@@ -0,0 +1,58 @@
+# mautrix-signal - A Matrix-Signal puppeting bridge
```

Comment:
neither of the functions in this file are used outside of `commands/signal.py`. There doesn't seem to be any reason to have them in a separate util module.

Reply:
done